frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-item-details.tsx

```diff
@@ -69,43 +78,47 @@ const InstalledHintBlock: React.FC<OperatorHubItemDetailsHintBlockProps> = ({
   const to = installedCSV
     ? `${nsPath}/clusterserviceversions/${installedCSV?.metadata?.name ?? ''}`
     : `${nsPath}/subscriptions/${subscription.metadata.name ?? ''}`;
+  const installedVersion = installedCSV?.spec?.version;
   return (
     <HintBlock className="co-catalog-page__hint" title="Installed Operator">
       <p>
-        {installedCSV && installedCSV?.spec?.version !== latestVersion ? (
+        {installedVersion !== latestVersion ? (
           <span>
-            Version <strong>{installedCSV?.spec?.version}</strong> of this operator has been
-            installed on the cluster.
+            <Trans ns="olm">
+              Version <strong>{{ installedVersion }}</strong> of this Operator has been installed on
+              the cluster.
+            </Trans>
           </span>
         ) : (
-          'This Operator has been installed on the cluster.'
+          t('olm~This Operator has been installed on the cluster.')
         )}
         &nbsp;
-        <Link to={to}>View it here.</Link>
+        <Link to={to}>{t('olm~View it here.')}</Link>
       </p>
     </HintBlock>
   );
 };
 
 const OperatorHubItemDetailsHintBlock: React.FC<OperatorHubItemDetailsHintBlockProps> = (props) => {
+  const { t } = useTranslation();
   const { installed, providerType } = props;
   if (installed) {
     return <InstalledHintBlock {...props} />;
   }
 
   if (providerType === 'Community') {
     return (
-      <HintBlock className="co-catalog-page__hint" title="Community Operator">
+      <HintBlock className="co-catalog-page__hint" title={t('olm~Community Operator')}>
         <p>
-          This is a community provided operator. These are operators which have not been vetted or
-          verified by Red Hat. Community Operators should be used with caution because their
-          stability is unknown. Red Hat provides no support for Community Operators.
+          {t(
+            'olm~This is a community provided Operator. These are Operators which have not been vetted or verified by Red Hat. Community Operators should be used with caution because their stability is unknown. Red Hat provides no support for Community Operators.',
```

Comment:
```suggestion
            'olm~This is a community provided Operator. These are Operators which have not been vetted or verified by Red Hat. Community Operators should be used with caution because their stability is unknown. Red Hat provides no support for community Operators.',
```